src/app/shared/components/route-selector/route-selector.component.ts

```
@@ -72,16 +72,16 @@ export class RouteSelectorComponent implements OnInit, ControlValueAccessor, OnD
 
   onChangeSub: Subscription;
 
-  form: FormGroup = this.fb.group({
-    mileageLocations: new FormArray([]),
+  form: UntypedFormGroup = this.fb.group({
+    mileageLocations: new UntypedFormArray([]),
     distance: [, Validators.required],
```

Biome (1.9.4)

[error] This array contains an empty slot.

Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)